wdl2cwl/main.py

```diff
@@ -997,7 +567,7 @@ def get_cwl_task_outputs(
                     )
                 )
             else:
-                glob_expr = self.get_expr(wdl_output)
+                glob_expr = get_expr(wdl_output, self.ctx)
                 glob_str = f"$({glob_expr})"
 
                 if wdl_output.type.optional:
```

Comment:
`main.py` has `1056` lines on `main` branch. This PR reduces it to `627`. So reading & maintaining `main.py` should become a bit easier.